collect_app/src/main/java/org/odk/collect/android/widgets/utilities/GeoPolyDialogFragment.kt

```diff
@@ -20,18 +21,22 @@ class GeoPolyDialogFragment(viewModelFactory: ViewModelProvider.Factory) :
             GeoPolyFragment.REQUEST_GEOPOLY,
             this
         ) { _, result ->
-            val result = result.getString(GeoPolyFragment.RESULT_GEOPOLY)
-            if (result != null) {
-                onAnswer(StringData(result))
+            val geopolyChange = result.getString(GeoPolyFragment.RESULT_GEOPOLY_CHANGE)
+            val geopoly = result.getString(GeoPolyFragment.RESULT_GEOPOLY)
+            val incremental = FormEntryPromptUtils.getBindAttribute(prompt, "incremental")
```

Comment:
I think we have consts form the supported bind attributes. Maybe a separate issue for that?

Reply:
Yes